docs/api/storage/SDBlockDevice.md

```diff
@@ -6,6 +6,17 @@ You can use the Mbed OS SD card block device, so applications can read and write
 
 To configure this class, please see our [BlockDevice configuration documentation](../reference/storage.html#blockdevice-default-configuration).
 
+## How to wire an external SD-card module to a developer board
+
+<span class="images">![](../images/sd-wiring.png)<span>How to wire an external SD-card module to a developer board</span></span>
```

Comment:
Should this PR also include the .png or how does that work?

Reply:
I added the PR separately: https://github.com/ARMmbed/mbed-os-5-docs/tree/development/docs/images

Reply:
Ok, a bit unclear why you would do those in seperate PRs.